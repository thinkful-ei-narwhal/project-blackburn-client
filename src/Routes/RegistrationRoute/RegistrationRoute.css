.Registration {
  background-color: #302b28;
  position: fixed;
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
}
.reg-container {
  margin: 25vh;
  padding: 20px;
  text-align: center;
  border-radius: 10px;
  background-color: #d9c1aa;
  box-shadow: 3px 3px 3px lightgray;
}
.reg-account {
  display: flex;
  align-items: center;
  flex-direction: column;
  justify-content: space-around;
  margin: 10px auto;
  padding: 20px;
  width: 50%;
}
.reg-input,
.reg-select {
  margin: 10px;
  padding: 5px;
  border: 1px solid #d9c1aa;
  border-radius: 10px;
}

.reg-btn {
  margin: 10px auto;
  margin-left: 10px;
  margin-right: 10px;
  padding: 20px;
  border: 1px solid #d9c1aa;
  border-radius: 6px;
  color: #d9c1aa;
  background-color: #302b28;
}
.reg-btn:hover {
  background-color: #ae9172;
  color: #302b28;
}
.reg-btn:disabled {
  display: none;
  pointer-events: none;
}

.error {
  color: red;
  font-size: 15px;
  font-weight: normal;
}
