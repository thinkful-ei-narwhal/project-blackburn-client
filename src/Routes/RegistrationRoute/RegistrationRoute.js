import React, { Component } from 'react';
import './RegistrationRoute.css';
import RegistrationForm from '../../Components/RegistrationForm/RegistrationForm';

class RegistrationRoute extends Component {
  render() {
    return (
      <div className="Registration">
        <h2>Create New Account:</h2>
        <RegistrationForm {...this.props} />
      </div>
    );
  }
}

export default RegistrationRoute;
