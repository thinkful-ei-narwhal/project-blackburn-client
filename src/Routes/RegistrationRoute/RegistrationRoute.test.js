import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import BlackBurnContext from '../../Context/BlackburnContext';
import RegistrationRoute from './RegistrationRoute';

describe(`Renders registration page to dom without crashing`, () => {
    it('renders checkpoint win crashing', () => {
        const wrapper = shallow(
        <BlackBurnContext.Provider>
            <RegistrationRoute />
        </BlackBurnContext.Provider>
    )
    .find('.Registration')
    expect(toJson(wrapper)).toMatchSnapshot()
    })
})