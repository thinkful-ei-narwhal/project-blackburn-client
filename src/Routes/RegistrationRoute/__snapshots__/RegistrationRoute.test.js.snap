// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders registration page to dom without crashing renders checkpoint win crashing 1`] = `null`;
